### DISCUSSION TITLE
Personal Access Token is not working

### DISCUSSION THREAD

**Discussion by patra24** [community/community#131182] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Other

### Body

The access token is not working. My command prompt is throwing an error as an invalid credential.

**Comment by noahstoner** [comment]
Hello.
Can you provide your screenshot?

**Comment by patra24** [comment]
I want to clone my repositories to Ed discussion environment, when I was asked to enter my username and password, I am getting the below error:
![Screenshot 2024-07-05 213539](https://github.com/community/community/assets/151497157/eba7fd15-fe32-4653-a660-cce06d625e71)

**Reply by noahstoner** [reply]
What's your system?

**Comment by gaomingzhao666** [comment]
Hey bro.

**In my opinion, there possibly are two reasons for this error.**

- It is probably invalided or deleted by GitHub when you push or pull any sensitive information.
- It is probably just because you wrong setting on `Personal Access Token`, such as you forgot to set the `write and reading` permission on this activity.

**You should check the permission on the `GitHub Setting`**

**Comment by Er-Sadiq** [comment]
Well it could be because the following
conflicting remote Repo Address
or the token has expired
you can delete the old token and create a new one try pushing it again that always help

**Comment by davevad93** [comment]
Hi @patra24 ,

If you want to do git operations with `HTTPS` you need a `PAT` with the `repo` scope checked, otherwise you'll be unable to `clone` or `push`:

![0](https://github.com/community/community/assets/94308234/52c400e2-44d1-42e1-ba94-11bc7136d9e4)

You can also do such operations with `SSH`, in that case the password used for authentication would be the one you've set when generating your `SSH Key`, for more info take a look at this from the docs:

- [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
- [Adding a new SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @patra24 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
